Modules/IO/GDCM/src/itkGDCMImageIO.cxx

```diff
@@ -302,18 +313,6 @@ GDCMImageIO::Read(void * pointer)
     image = ialut.GetOutput();
     len *= 3;
   }
-  else if (pi == gdcm::PhotometricInterpretation::YBR_FULL ||
-           pi == gdcm::PhotometricInterpretation::YBR_FULL_422)
-  {
-    // ITK does not carry color space associated with an image. It is pretty
-    // much assumed that color image is expressed in RGB.
-    gdcm::ImageChangePhotometricInterpretation icpi;
```

Comment:
By removing the use of `gdcm::ImageChangePhotometricInterpretation` you are making it hard to support conversion from `MONOCHROME1` to `MONOCHROME2`.

Reply:
> You should really start using the correct filter: gdcm::ImageChangePhotometricInterpretation

No. Sorry.

Reply:
It doesn't affect monochrome at all at this point. Sorry.

Reply:
?????